core/profiling/profiling.h

```diff
@@ -101,4 +107,6 @@ void godot_init_profiler();
 #define GodotProfileZoneGroupedEndEarly(m_group_name, m_zone_name)
 #define GodotProfileZoneGrouped(m_group_name, m_zone_name)
 
+#define GodotProfileAlloc(m_ptr, m_size)
+#define GodotProfileFree(m_ptr)
```

Comment:
After the merge of https://github.com/godotengine/godot/pull/112725, these two should get similar comments now.